lte/gateway/python/magma/pipelined/rpc_servicer.py

```diff
@@ -1042,7 +1042,8 @@ def _ng_qer_update(
                 self._ng_inactivate_qer_flows(ipv6, qos_enforce_rule)
 
         # Install PDR rules
-        elif pdr_entry.pdr_state == PdrState.Value('INSTALL'):
+        elif pdr_entry.pdr_state == PdrState.Value('INSTALL') or \
+                    pdr_entry.pdr_state == PdrState.Value('MODI'):
```

Comment:
For Modification Case, we need to put extra check like ``` if qos_enforce_rule.polices.policies:``` before process the install the rules.